packages/lending/src/protocols/radiant-v2/configs.ts

```diff
@@ -29,6 +29,13 @@ export const configs: Config[] = [
       PriceOracle: '0x0BB5c1Bc173b207cBf47CDf013617087776F3782',
     },
   },
+  {
+    chainId: common.ChainId.base,
+    contractMap: {
+      ProtocolDataProvider: '0x07d2DC09A1CbDD01e5f6Ca984b060A3Ff31b9EAF',
+      PriceOracle: '0xe373749cd9b2D379f7f6Dd595e5164498b922164',
```

Comment:
https://docs.radiant.capital/radiant/contracts-and-security/base-contracts#miscellaneous